spec/Order.md

```diff
@@ -0,0 +1,58 @@
+CodeChain supports an on-chain order to support DEX(Decentralized Exchange).
+
+## Order
+
+An order can be either a _point-to-point order_ between a maker and a taker, or a _broadcast order_ between a maker, a relayer and takers.
+Point-to-point orders allow two parties(makers and takers) to directly exchange assets between each other.
+Broadcast orders are similar to point-to-point orders, but also allows relayers to take fee assets from makers and takers. Broadcast orders are usually used in decentralized exchange platforms.
```

Comment:
Broadcast orders are similar to point-to-point orders, but also allows relayers to take fee assets from makers and takers. Broadcast orders are usually used in decentralized exchange platforms. -> Broadcast orders are similar to point-to-point orders, but also allow relayers to take fee assets from makers and takers. Broadcast orders are usually used in decentralized exchange platforms.